src/convertToObject.js

```diff
@@ -6,7 +6,26 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styleObject = {};
+  const declarations = sourceString.split(';');
+
+  declarations.forEach((declaration) => {
+    const newDeclaration = declaration.trim();
+
+    if (!declaration) {
+      return;
+    }
+
+    const [property, value] = newDeclaration
```

Comment:
The split by colon ':' does not account for the possibility of colons appearing in the property values. While not required for this task, handling such cases could make the function more reliable.